flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/java/org/apache/flink/cdc/connectors/mysql/source/assigners/MySqlSnapshotSplitAssignerTest.java

```diff
@@ -574,9 +576,12 @@ private List<String> getTestAssignSnapshotSplitsFromCheckpoint(AssignerStatus as
         List<TableId> alreadyProcessedTables = new ArrayList<>();
         alreadyProcessedTables.add(processedTable);
 
+        Properties jdbcProperties = new Properties();
+        jdbcProperties.put(PropertyKey.tinyInt1isBit.getKeyName(), "true");
         RowType splitKeyType =
                 ChunkUtils.getChunkKeyColumnType(
-                        Column.editor().name("id").type("INT").jdbcType(4).create());
+                        Column.editor().name("id").type("INT").jdbcType(4).create(),
+                        jdbcProperties);
```

Comment:
Please add some other tests for this part. A table with boolean  + tinyint(1) + tinyint(2) columns will behavior differently with tinyInt1isBit = false or true.